.github/workflows/release.yaml

```diff
@@ -0,0 +1,74 @@
+name: Build and Push Database Exporter
+
+on:
+  push:
+    tags:
+      - "*.*.*-*"
+
+env:
+  UBUNTU_VERSION: '22.04'
+  RUST_IMAGE_TAG: '1.73.0'
+  OUTFILE: "s3://eur2ccd.concordium.com/concordium-eur2ccd_${{ github.ref_name }}_amd64.deb"
+  IAM_ROLE: "arn:aws:iam::192549843005:role/github_eur2ccd" 
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  release-euro2ccd:
+    runs-on: ubuntu-latest
+    environment: release
+    steps:
+      - name: Get the eur2ccd version
+        env:
+          TAG: ${{ github.ref_name }}
+        run: echo "EUR2CCD_VERSION=${TAG%%-*}" >> $GITHUB_ENV
+
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref_name }}
+          submodules: recursive
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: "eu-west-1"
+          role-to-assume: ${{ env.IAM_ROLE }}
+          role-session-name: ReleaseEur2ccdSession
+      
+      - name: Get version from cargo.toml
+        run: |
+          VERSION=$(yq .package.version Cargo.toml)
+          echo "::notice FULL_IMAGE_TAG=${FULL_IMAGE_TAG}"
+          # Make sure the image does not exist. Abort if we can retrieve any metadata.
+          if [ ! "${{ env.EUR2CCD_VERSION }}" = "$VERSION" ]; then
+             echo "::error ${{ github.ref_name }} does not match ${{ env.SERVICE_NAME }}/${VERSION}."
+             exit 1
+          fi
+
+      - name: Check if version exist in s3
+        run: |
+          set +e
+          totalFoundObjects=$(aws s3 ls "${{env.OUTFILE}}")
+          ec=$?
+          if [ $ec -eq "0" ]; then
+              echo "Following matching objects found:"
+              echo $totalFoundObjects
+              exit 1
+          elif [ $ec -eq "255"]; then
+              echo "error: You do not have aws access"
```

Comment:
Just write the output here instead. It may also be because of for example no connection.